### PR TITLE
ADCM-6126 Mapping management

### PR DESCRIPTION
**Pull request by Sealwing**

### CODE REVIEW

File: adcm_aio_client/core/objects/cm.py [hunk 1]
```diff
@@ -135,8 +134,8 @@ async def set_ansible_forks(self: Self, value: int) -> Self:
     # nodes and managers to access
 
     @cached_property
```
Comment (DanBalalan) [inline adcm_aio_client/core/objects/cm.py:136]: `@async_cached_property`?

Comment (Sealwing) [reply adcm_aio_client/core/objects/cm.py:136]: yep, thanks

File: adcm_aio_client/core/actions/_objects.py [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    async def _rich_data(self: Self) -> dict:
+        return (await self._requester.get(*self.get_own_path())).as_dict()
+
+
+class ActionsAccessor(NonPaginatedChildAccessor):
+    class_type = Action
+
+
+async def detect_cluster(owner: InteractiveObject) -> Cluster:
+    from adcm_aio_client.core.objects.cm import Cluster, Component, Host, Service
```
Comment (Starovoitov) [inline adcm_aio_client/core/actions/_objects.py:70]: is it circular import there? why to import here?

File: adcm_aio_client/core/actions/_objects.py [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+
+from asyncstdlib import cached_property as async_cached_property
+
+from adcm_aio_client.core.errors import HostNotInClusterError, NoMappingRulesForActionError
+from adcm_aio_client.core.mapping import ActionMapping
+from adcm_aio_client.core.objects._accessors import NonPaginatedChildAccessor
+from adcm_aio_client.core.objects._base import InteractiveChildObject, InteractiveObject
+
+if TYPE_CHECKING:
+    from adcm_aio_client.core.objects.cm import Cluster
```
Comment (Starovoitov) [inline adcm_aio_client/core/actions/_objects.py:14]: why Cluster is imported twice?

File: adcm_aio_client/core/actions/_objects.py (hunk 2 above)
Comment (Sealwing) [reply adcm_aio_client/core/actions/_objects.py:70]: yes, circular import is in play here

File: adcm_aio_client/core/actions/_objects.py (hunk 3 above)
Comment (Sealwing) [reply adcm_aio_client/core/actions/_objects.py:14]: because it's imported for type checking (in this case there's no circular imports, I believe)

**Review by DanBalalan — APPROVED** [review]

**Review by a-alferov — APPROVED** [review]

**Pull request merged**